flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/logical/operators.scala

```diff
@@ -56,6 +57,11 @@ case class Project(projectList: Seq[NamedExpression], child: LogicalNode) extend
             case gcf: GetCompositeField => Alias(gcf, gcf.aliasName().getOrElse(s"_c$i"))
             case other => Alias(other, s"_c$i")
           }
+          /**
+            * when TableFunction.apply returns a Table,
+            * ex: t1.join(split('a) as 'b), there comes the new case
+            * */
+          case alias: Alias => alias
```

Comment:
// This happens when TableFunction.apply returns a Table, ex: t1.join(split('a) as 'b)